src/components/productsList/_productsList.scss

```diff
@@ -0,0 +1,121 @@
+$media-tablet: 768px;
+$media-desktop: 1024px;
+$padding-mobile: 20px;
+$padding-desktop: 40px;
+
+.products-list {
+  display: flex;
+  flex-direction: column;
+  margin: 40px 10px 10px;
+
+  * {
+    box-sizing: border-box;
+  }
+
+  @media screen and (min-width: $media-desktop) {
+    flex-direction: row;
+  }
+}
+
+.products-list-nav {
+  $self: &;
```

Comment:
Nice! Neat trick right?

Reply:
Hehehe, like the oldschool self=this in javascript :)